Tools/Schema.NET.Tool/SchemaSourceGenerator.cs

```diff
@@ -95,174 +141,271 @@ private static string RenderClass(GeneratorSchemaClass schemaClass)
+                }
+
+                bool success;
+            {{conditions}}
+                else
+                {
+                    success = base.TrySetValue(property, value);
+                }
+
+                return success;
+            }
+            """,
+            indent);
     }
 
-    [SuppressMessage(
-        "StyleCop.CSharp.LayoutRules",
-        "SA1513:Closing brace should be followed by blank line",
-        Justification = "Interpolated string")]
+    private static string RenderClassTryGetVaue1(GeneratorSchemaProperty[] allProperties, int indent)
+    {
+        if (allProperties.Length == 0)
+        {
+            return SourceUtility.Render(
+                """
+                /// <inheritdoc/>
+                public override bool TryGetValue<TValue>(string property, out OneOrMany<TValue> result)
+                {
+                    if (string.IsNullOrWhiteSpace(property))
+                    {
+                        result = default;
+                        return false;
+                    }
+
+                    return base.TryGetValue(property, out result);
+                }
+                """,
+                indent);
+        }
+
+        var conditions = SourceUtility.RenderItems(
+            allProperties,
+            (index, indent, property) =>
+            $$"""
+            {{(index == 0 ? "if" : "else if")}} ("{{property.Name}}".Equals(property, StringComparison.OrdinalIgnoreCase))
+            {
+            {{SourceUtility.RenderItems(
+                property.CSharpTypes,
+                (index, indent, propertyType) =>
+                $$"""
+                {{(index == 0 ? "if" : "else if")}} (typeof({{propertyType}}) == typeof(TValue))
+                {
+                    result = (OneOrMany<TValue>)(IValues)this.{{property.Name}}{{(property.CSharpTypes.Count > 1 ? $".Value{index + 1}" : string.Empty)}};
+                    success = true;
+                }
+                """,
+                indent)}}
+                else
+                {
+                    result = default;
+                }
+            }
+            """,
+            indent,
+            SourceDelimeter.NewLine);
+
+        return SourceUtility.Render(
+            $$"""
+            /// <inheritdoc/>
+            public override bool TryGetValue<TValue>(string property, out OneOrMany<TValue> result)
+            {
+                if (string.IsNullOrWhiteSpace(property))
+                {
+                    result = default;
+                    return false;
+                }
+
+                var success = false;
+            {{conditions}}
+                else
+                {
+                    success = base.TryGetValue(property, out result);
+                }
+
+                return success;
+            }
+            """,
+            indent);
+    }
+
+    private static string RenderClassTryGetVaue2(GeneratorSchemaProperty[] allProperties, int indent)
```

Comment:
Typo in method name - thinking maybe it would be better called `RenderClassTryGetValue_IValues`.

Reply:
Done